README.md

```diff
@@ -208,10 +208,13 @@ Available targets:
 | existing\_workers\_role\_policy\_arns | List of existing policy ARNs that will be attached to the workers default role on creation | `list(string)` | `[]` | no |
 | existing\_workers\_role\_policy\_arns\_count | Count of existing policy ARNs that will be attached to the workers default role on creation. Needed to prevent Terraform error `count can't be computed` | `number` | `0` | no |
 | id\_length\_limit | Limit `id` to this many characters.<br>Set to `0` for unlimited length.<br>Set to `null` for default, which is `0`.<br>Does not affect `id_full`. | `number` | `null` | no |
-| instance\_types | Set of instance types associated with the EKS Node Group. Defaults to ["t3.medium"]. Terraform will only perform drift detection if a configuration value is provided | `list(string)` | n/a | yes |
+| instance\_type | Instance type associated with the EKS Node Group. Defaults to "t3.medium". Terraform will only perform drift detection if a configuration value is provided | `string` | n/a | yes |
 | kubernetes\_labels | Key-value mapping of Kubernetes labels. Only labels that are applied with the EKS API are managed by this argument. Other Kubernetes labels applied to the EKS Node Group will not be managed | `map(string)` | `{}` | no |
 | kubernetes\_version | Kubernetes version. Defaults to EKS Cluster Kubernetes version. Terraform will only perform drift detection if a configuration value is provided | `string` | `null` | no |
 | label\_order | The naming order of the id output and Name tag.<br>Defaults to ["namespace", "environment", "stage", "name", "attributes"].<br>You can omit any of the 5 elements, but at least one must be present. | `list(string)` | `null` | no |
+| launch\_template\_id | The ID of a custom launch template to use for the EKS node group. | `string` | `null` | no |
```

Comment:
We can also provide a `launch_template.name` based on the docs, do we only want to allow `id` instead?

Reply:
Yeah, I saw that, although the `id` seemed definitive enough to be honest. I don't know if I like the idea of adding the template just by the name, but I'll chew on this a bit.

Reply:
Honestly the opinionated view of only allowing id doesn't seem all that bad at all, it's definitely intuitive